README

```diff
@@ -22,56 +22,60 @@ list is available below.
 
 [source, console]
 ----
-$ sudo apt install libpam-u2f pamu2fcfg
```

Comment:
> Despite my preference for the conventional use of the pseuod-prompts (`$` and
`#`) I think they should be removed, as they end up in the way for those users
who like to copy-paste commands to their terminals. 

I think we should probably try to stay consistent across our projects (libfido2, yubihsm-shell, python-yubihsm, yubikey-manager, ..., all use this style) and keep these prompt symbols. 

> Moreover, Github seems to
interpret the latter as shell comment, and renders `#`-prefixed lines as
comments.

We've historically had some issues with the rendering also for developers.yubico.com.  What do you think of simply making these a literal block instead of a source block?

Reply:
Acceptable. I like it consistent.

Reply:
Following up given the recent changes: `[source]` is especially problematic with d.y.c. Without a specified language, [it will (or at least did historically) render invisible source blocks](https://github.com/yubico/pam-u2f/commit/fc01804deffe4658f75cc8fcebc3aabf155a013c)!

I believe also dropping `[source]` will work or dropping the delimiters and prefixing the lines with spaces instead, but we may want to manually build d.y.c. locally to verify.

Reply:
Dropping the whole tag does the job:

![image](https://github.com/user-attachments/assets/2d630c1d-5be6-42c6-8add-95b6ad2fd3b0)